packages/client-node/integration/LegalOfficer.ts

```diff
@@ -19,3 +20,33 @@ export async function workload(state: State) {
         expect(workload).toBe(0);
     }
 }
+
+export async function updateLegalOfficer(state: State) {
+    const { client, alice } = state;
+    const updatedAlice: LegalOfficer = {
+        node: alice.node,
+        userIdentity: {
+            firstName: "Alice",
+            lastName: "updated-last-name",
+            email: "updated-email-address@logion.network",
+            phoneNumber: "updated-phone-number",
+        },
+        postalAddress: {
+            company: "updated-company",
+            line1: "updated-line1",
+            line2: "updated-line2",
+            postalCode: "updated-postal-code",
+            city: "updated-city",
+            country: "updated-country",
+        },
+        account: alice.account,
+        additionalDetails: "Some new details",
+    } as LegalOfficer;
+    const aliceClient = client.withCurrentAccount(alice.account);
+    await aliceClient.directoryClient.createOrUpdate(updatedAlice);
+    const legalOfficers = await client.directoryClient.getLegalOfficers();
```

Comment:
As there is no more directory, the client should probably be renamed.

Reply:
Done.